src/lib/SapphireClient.ts

```diff
@@ -6,13 +6,19 @@ import type { Plugin } from './plugins/Plugin';
 import { PluginManager } from './plugins/PluginManager';
 import { ArgumentStore } from './structures/ArgumentStore';
 import { CommandStore } from './structures/CommandStore';
+import { InteractionHandlerStore } from './structures/InteractionHandlerStore';
 import { ListenerStore } from './structures/ListenerStore';
 import { PreconditionStore } from './structures/PreconditionStore';
 import { BucketScope, PluginHook } from './types/Enums';
 import { Events } from './types/Events';
+import { acquire } from './utils/application-commands/ApplicationCommandRegistries';
 import { ILogger, LogLevel } from './utils/logger/ILogger';
 import { Logger } from './utils/logger/Logger';
 
+container.applicationCommandRegistries = { acquire };
+
+const optionalListenersPath = join(__dirname, '..', 'optional-listeners');
```

Comment:
Would be better as a `#private` property imo

Reply:
Brings absolutely no benefits soooo 🤔

Reply:
I know it doesn't, it just feels against object oriented paradigms to have a file whose runtime code is basically all a class and then 1 constant above it.

Reply:
Also, this constant lives forever and isn't GC'd, which might be undesirable.